mpcforces_extractor/frontend/templates/mpcs.html

```diff
@@ -48,8 +54,16 @@ <h1 id="mpcs-title" style="cursor: pointer;">MPCs</h1>
                     <th>Config</th>
                     <th>Master</th>
                     <th>Slaves</th>
-                    <th>Part Id to Nodes</th>
```

Comment:
bothers me that we dont have partid2nodeid ... maybe replace the slave cell with partid2node ids copy buttons